### ISSUE TITLE
Alelec.net is down? 

### ISSUE THREAD

**Issue by Naphier** [andrewleech/kodi_repo#1]
Hi. Can't seem to resolve any URL under alelec.net

Is there an alternative?

**Comment by andrewleech** [comment]
Hi, I didn't think anyone would be using anything from my repo and the server was pretty inefficient so I shut it down. 

What were you using from it?

**Comment by Naphier** [comment]
It's referenced in a lot of tutorials on adding Netflix to Kodi. That's
where I found it from. There's a Netflix XMBC plugin for Kodi. Thought I
just used it a week ago. Did you just recently shut down the server?

**Comment by andrewleech** [comment]
Oh that's surprising. I had a Netflix plugin that stopped working almost 3 years ago after Netflix changed their site... I wouldn't have expected it to magically start working again...
I'm pretty sure it's a few weeks since I stopped my repo server when it's CPU use was a bit high, I'll start it again now.

**Comment by pgirolami** [comment]
I confirm your repo is referenced in tons of articles about getting Netflix on Kodi for raspberry pi. For example https://www.ivacy.com/blog/how-to-install-kodi-netflix-addon/ references http://www.alelec.net/kodi/repository.alelec.zip but I still can't access it.
Did you start it back up ?

**Comment by andrewleech** [comment]
Sorry to say but that "tutorial" is just a vpn provider copying a three year old tute from somewhere else to convince you to buy their vpn. 
My old Netflix plugin certainly still want working a few months ago: https://forum.kodi.tv/showthread.php?tid=211574&page=102

I tried to restart my repo server but clearly it's still not working. However if you just want it for the Netflix plugin you'll be disappointed, it's not going to work. 

This looks like it might be working though? https://github.com/yamboo-efi/plugin.video.flix2kodi
I haven't tried it myself (don't have Netflix subscription)

**Comment by Naphier** [comment]
Yeah I read a few forum posts and unfortunately those VPN tutorials are the first few things that come up. I was mistaken before thinking the I used the alelec repo. I was installing Netflix on OMSC before. Apparently for Kodi we need version 18 and then there's a netflix plugin that works. Unfortunately I've not had time to try that out.
Those VPS turds are just re-dating their posts to make them look new. Jerks.

**Comment by noci2012** [comment]
alelec.net  is just redirecting  to itself  until the limit for redirects is reached.

**Comment by andrewleech** [comment]
Yes, sorry my kodi repository is not currently running. Was there anything in particular you're looking for?

**Comment by Free2live** [comment]
Hi mate. I used your Netflix addon last year on a raspberry pi 3 and it was working. Call it magic ;)
